packages/i18n/en.json

```diff
@@ -39,6 +39,10 @@
     }
   },
   "pages": {
+     "404": {
+      "title": "Whooops ?!… sounds you're lost… Sorry. Really.",
```

Comment:
Can you add a title and a alternativeTitle like in LostPassword, for exemple :
```
"title": "Page not found",
"alternativeTitle": "Whooops ?!… sounds you're lost…",
```
to use this method in page 404
```js
useHead({
  titleTemplate: `${t('pages.404.title')} | %s`,
})
```
which allow to update title in navigator's tab

Reply:
done